contracts/Converter.sol

```diff
@@ -91,6 +91,9 @@ contract Converter is Ownable, ReentrancyGuard {
         IPolydexPair pair = IPolydexPair(factory.getPair(token0, token1));
 
         require(address(pair) != address(0), "Invalid pair");
+        IPolydexPair wmatic_pair0 = IPolydexPair(factory.getPair(token0, wmatic));
+        IPolydexPair wmatic_pair1 = IPolydexPair(factory.getPair(token1, wmatic));
```

Comment:
Remove this from here. Put this check inside _toWMatic() function on line no 148 - 150.